timesketch/frontend-ng/package.json

```diff
@@ -29,6 +29,7 @@
     "vue-dragscroll": "^3.0.0",
     "vue-moment": "^4.0.0",
     "vue-router": "^3.0.2",
+    "vue-shortkey": "^3.1.7",
```

Comment:
This plugin/library is not compatible with Vue3, and doesn't seem to be maintained (latest release is from 2018).

I opt to refactor this and write our own component for keyboard shortcuts instead.